### PR TITLE
Update version on master

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

**Comment by schiessle** [comment]
as said in the other issue, let's keep it 14 to 16. I will always create a branch if we introduce incompatibilities to the server, otherwise the app can be decoupled from the server version.

**Comment by MorrisJobke** [comment]
integration tests fail

File: appinfo/info.xml [hunk 1]
```diff
@@ -16,15 +16,15 @@ The following providers are supported and tested at the moment:
...
+	<version>2.1.0</version>
 	<licence>agpl</licence>
 	<author>Lukas Reschke</author>
 	<namespace>User_SAML</namespace>
 	<types>
 		<authentication/>
 	</types>
 	<documentation>
-		<admin>https://docs.nextcloud.com/server/13/admin_manual/configuration_server/sso_configuration.html</admin>
+		<admin>https://docs.nextcloud.com/server/16/admin_manual/configuration_server/sso_configuration.html</admin>
```
Comment (nickvergessen) [inline appinfo/info.xml]: @schiessle also use a go.php link here please

**Comment by schiessle** [comment]
> integration tests fail

seems to be again an issue with the LDAP docker... This problem already existed a few days ago, then magically disappeared yesterday and now is back again :confused:

**Pull request merged**